### DISCUSSION TITLE
Tags page is not loaded

### DISCUSSION THREAD

**Discussion by ullas-jain** [mmistakes/minimal-mistakes#2981] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/54089173/119935008-65e54c80-bfa4-11eb-98df-479ee4fbbc1b.png)

Pretty sure this might be a configuration issue. I have been reading the documentation, I couldn't find the solution for it yet.
Hence posting here 🙏 

## Environment

- Minimal Mistakes version: 4.23.0
- Ruby gem or remote theme version:
- Jekyll version:
- Git repository URL: https://github.com/ullas-jain/ullas-jain.github.io
- GitHub Pages hosted (if yes provide URL to site): https://ullas-jain.github.io/
- Operating system: Mac OSX

## Expected behavior

Tags should be loaded

## Steps to reproduce the behavior

- Forked the repository
- Added `_posts` folder with couple of posts including tags
- Configured the `config.yml` according to documentation.
- Tags is not being loaded

https://ullas-jain.github.io/tags/#dependencies

**Comment by iBug** [comment, marked as answer]
<https://mmistakes.github.io/minimal-mistakes/docs/layouts/#taxonomy-archives>

> If you’re not using the `jekyll-archives` plugin then you need to create archive pages yourself.

**Comment by ullas-jain** [comment]
Thanks @iBug. I'll check that.
